frontend_app_workspace_advanced/src/component/WorkspaceAdvancedConfiguration.jsx

```diff
@@ -74,7 +74,7 @@ export class WorkspaceAdvancedConfiguration extends React.Component {
               trigger={isMobile ? 'focus' : 'hover'}
             >
               <PopoverBody>
-                {props.t('This is the role that members will have by default when they join your space.')}
+                {props.t('This is the role that members will have by default when they join your space if it is open or on request.')}
```

Comment:
Shouldn't we just hide this message if the selected space is not open or on request?
adding this if clause to this message makes it less easy to understand I think

Reply:
All functionality will have to be hidden not only the message, it goes away from the objective of this issue. Don't hesitate to create an issue for it, but I believe the idea was to evolve the functionality to be used in confidential spaces as well (to be discussed with the product manager).

Reply:
then can I suggest to:
 - not update this sentence for now, or
 - use something like "This is the role that members will have by default when they join your space (for open and on request spaces only)"?

Reply:
changed